src/components/Pages/Guide/index.js

```diff
@@ -151,7 +154,17 @@ function Guide(props) {
           relatedDepartments={relatedDepartments}
         />
       )}
+
       {image && <PageBanner image={image} />}
+
+      {!image &&
+        <img
+          className = "coa-GuidePage__guide-page-placeholder"
+          src={guidePagePlaceholder}
+          alt={intl.formatMessage(i18n.guidePagePlaceholder)}
+        />
+      }
+
```

Comment:
Here's where we'll handle the "has-an-image" || "doesnt-have-an-image" page. I put it here because within the `PageBanner` component, the image is being styles(more dynamically) than the static placeholder image.

Reply:
👍